simulation_parameters/microbe_stage/compounds.json

```diff
@@ -88,6 +88,22 @@
         },
         "iconPath": "res://assets/textures/gui/bevel/oxytoxy.png"
     },
+    "mucilage": {
+        "name": "MUCILAGE",
+        "volume": 1,
+        "isCloud": true,
+        "decayRate": 0.99,
+        "isAgent": true,
+        "isEnvironmental": false,
+        "canBeDistributed": true,
+        "digestible": true,
```

Comment:
Just to make sure, gaining mucilage from digested cells is intentional for gameplay right?

Reply:
Ah, I suppose I hadn't considered that. My gut feel is that it shouldn't be, but other opinions are welcome.

Reply:
So this should be changed to false here, right?

Reply:
Looks like I changed digestible for the wrong compound. I'll fix that.